db_postgres.go

```diff
@@ -89,6 +89,15 @@ var postgresMigrations = []string{
 			TYPE sasl_mechanism
 			USING sasl_mechanism::sasl_mechanism;
 	`,
+	`
+		CREATE TABLE "ReadReceipt" (
+			id SERIAL PRIMARY KEY,
+			network INTEGER NOT NULL REFERENCES "Network"(id) ON DELETE CASCADE,
+			target VARCHAR(255) NOT NULL,
+			timestamp INTEGER NOT NULL,
```

Comment:
`TIMESTAMP WITH TIME ZONE` should be used instead (and is compatible with `time.Time`).